### ISSUE TITLE
Input Binding for Azure SQL

### ISSUE THREAD

**Issue by paulyuk** [dapr/components-contrib#913]
In what area(s)?

Describe the feature
To be able to trigger off of Azure SQL data.  

Release Note
RELEASE NOTE:

**Comment by artursouza** [comment]
I like this idea. The only concern is if Azure SQL would offer an API to notify back to the app on data changes.

**Comment by yaron2** [comment]
Why does this need to be tied to Azure SQL? surely SQL Server has triggers.

**Comment by paulyuk** [comment]
Agreed this makes sense for more SQL databases.  The first request started with Azure SQL.

**Comment by CodeMonkeyLeet** [comment]
@paulyuk Can you adjust the priority on this as well if the request may no longer be P1?
